tests/unittests/MLTest.cpp

```diff
@@ -62,12 +70,12 @@ TEST_P(MLTest, learnSqrt2Placeholder) {
 
   bindings.allocate(SN->getPlaceholder());
 
-  Function *TF = glow::differentiate(F, TC);
-  EE_.compile(CompilationMode::Train, TF);
+  glow::differentiate(F, TC);
+  EET_.compile(CompilationMode::Train);
 
   // Train the network:
   for (int i = 0; i < 100; i++) {
-    EE_.run(bindings);
+    EET_.run(bindings, "Square root of 2_grad");
```

Comment:
nit: This is a little fragile, since we're assuming the naming won't change. Probably makes more sense to still keep the ptr to the differentiated function (i.e. `Function *TF = glow::differentiate(F, TC)` above) and then get its name before we call `compile()`, and then can just pass that name in here.